tests/panels/test_sql_geodjango_fix.py

```diff
@@ -0,0 +1,137 @@
+"""
+Tests for GeoDjango binary parameter handling fix
+"""
+
+import base64
+import json
+
+from debug_toolbar.panels.sql.decoders import DebugToolbarJSONDecoder
+from debug_toolbar.panels.sql.tracking import NormalCursorMixin
+
+from ..base import BaseTestCase
+
+
+class MockCursor:
+    """Mock cursor for testing"""
+
+
+class MockConnection:
+    """Mock database connection for testing"""
+
+    vendor = "postgresql"
+    alias = "default"
+
+
+class MockLogger:
+    """Mock logger for testing"""
+
+    def record(self, **kwargs):
+        pass
+
+
+class TestCursor(NormalCursorMixin):
+    """Test cursor that can be instantiated"""
+
+    def __init__(self):
+        self.cursor = MockCursor()
+        self.db = MockConnection()
+        self.logger = MockLogger()
+
+
+class GeoDjangoBinaryParameterTest(BaseTestCase):
+    """Test cases for GeoDjango binary parameter handling"""
+
+    def test_binary_parameter_encoding_decoding(self):
+        """Test that binary parameters are properly encoded and decoded"""
+        cursor = TestCursor()
+
+        # Test binary data similar to GeoDjango EWKB geometry
+        binary_data = b"\x01\x01\x00\x00\x20\xe6\x10\x00\x00\xff\xfe\xfd"
+        encoded = cursor._decode(binary_data)
+
+        self.assertIsInstance(encoded, dict)
+        self.assertIn("__djdt_binary__", encoded)
+
+        expected_b64 = base64.b64encode(binary_data).decode("ascii")
+        self.assertEqual(encoded["__djdt_binary__"], expected_b64)
+
+        json_params = json.dumps([encoded])
+        reconstructed = json.loads(json_params, cls=DebugToolbarJSONDecoder)
+
+        self.assertEqual(len(reconstructed), 1)
+        self.assertEqual(reconstructed[0], binary_data)
+        self.assertIsInstance(reconstructed[0], bytes)
```

Comment:
I don't think there's a benefit to being this verbose. Try to be more concise such as:

```suggestion
        self.assertEqual(reconstructed, [binary_data])
```